Description: Run libretime processes under www-data
 This allows the various processes to access logging and temporary files
 correctly.
Author: Kyle Robbertze <krobbertze@gmail.com>
Last-Update: 2018-12-15

---

Index: packaging/installer/systemd/airtime-liquidsoap.service
===================================================================
--- packaging.orig/installer/systemd/airtime-liquidsoap.service
+++ packaging/installer/systemd/airtime-liquidsoap.service
@@ -4,7 +4,7 @@ Description=Airtime Liquidsoap Service
 [Service]
 ExecStart=/usr/bin/airtime-liquidsoap
 User=libretime-playout
-Group=libretime-playout
+Group=www-data
 Restart=always
 
 [Install]
Index: packaging/installer/systemd/airtime-playout.service
===================================================================
--- packaging.orig/installer/systemd/airtime-playout.service
+++ packaging/installer/systemd/airtime-playout.service
@@ -4,7 +4,7 @@ Description=Airtime Playout Service
 [Service]
 ExecStart=/usr/bin/airtime-playout
 User=libretime-pypo
-Group=libretime-pypo
+Group=www-data
 Restart=always
 
 [Install]
Index: packaging/installer/systemd/airtime_analyzer.service
===================================================================
--- packaging.orig/installer/systemd/airtime_analyzer.service
+++ packaging/installer/systemd/airtime_analyzer.service
@@ -4,7 +4,7 @@ Description=LibreTime Media Analyzer Ser
 [Service]
 ExecStart=/usr/bin/airtime_analyzer
 User=airtime-analyzer
-Group=airtime-analyzer
+Group=www-data
 Restart=always
 
 [Install]
